apps/docs/content/docs/components/image.mdx

```diff
@@ -41,6 +41,7 @@ The Image component is used to display images with support for fallback.
 ## Usage
 
 <CodeDemo title="Usage" files={imageContent.usage} />
+> **Note**: The `<Image />` component is, by default, a `<div>` element containing an `<img />` child element. To use the properties of the `<img />` element directly, set `removeWrapper={true}`.
```

Comment:
> To use the properties of the `<img />` element directly, set `removeWrapper={true}`

this part is confusing. if i specify some classes to img, no matter with wrapper or not, the styles will be in img props instead of wrapper props.